guides/source/active_storage_overview.md

```diff
@@ -473,20 +473,25 @@ message.video.open do |file|
 end
 ```
 
-Transforming Images
--------------------
+Analyzing Files
+---------------
 
-To create a variation of the image, call `variant` on the `Blob`. You can pass
-any transformation to the method supported by the processor. The default
-processor is [MiniMagick](https://github.com/minimagick/minimagick), but you
-can also use [Vips](https://www.rubydoc.info/gems/ruby-vips/Vips/Image).
+Active Storage [analyzes](https://api.rubyonrails.org/classes/ActiveStorage/Blob/Analyzable.html#method-i-analyze) files after they've been uploaded by queuing a job in Active Job. Analyzed files will store additional information in the metadata hash, including `analyzed: true`. You can check whether a blob has been analyzed by calling `analyzed?` on it.
 
-To enable variants, add the `image_processing` gem to your `Gemfile`:
+To enable analysis, add the `image_processing` gem to your `Gemfile`:
```

Comment:
This is only necessary for image analysis. The video analyzer has a dependency on `ffmpeg`. You shouldn't need any gems to enable analysis generically.

Can we create separate sections for the provided image and video analyzers?